src/routes/new.jsx

```diff
@@ -139,6 +143,11 @@ export default function NewIssuePage() {
   const [submittedIssueUrl, setSubmittedIssueUrl] = useState("");
   const [projectData, setProjectData] = useState(null);
   const [accessError, setAccessError] = useState(null);
+  const navigate = useNavigate();
+
+  if (authState.type === "unauthenticated") {
+    navigate("/login");
+  }
```

Comment:
Instead of relying on state update for rendering the `<Login>`, we redirect to `/login`

Reply:
can we add an optional `?redirectTo` query parameter here?